### PR TITLE
Fix wrong table name User

### PR DESCRIPTION
**Pull request by jbartece**

### CODE REVIEW

File: model/src/main/java/org/jboss/pnc/model/User.java [hunk 1]
```diff
@@ -34,7 +34,7 @@
  * @author avibelli
  */
 @Entity
-@Table(uniqueConstraints = { @UniqueConstraint(name = "uk_user_email", columnNames = { "email" }),
+@Table(name = "UserTable", uniqueConstraints = { @UniqueConstraint(name = "uk_user_email", columnNames = { "email" }),
```
Comment (matejonnet) [inline model/src/main/java/org/jboss/pnc/model/User.java]: Why do we need to force new table name ?

Comment (jbartece) [inline model/src/main/java/org/jboss/pnc/model/User.java]: As User is restricted keyword for Postgres and deployment failed because of that

Comment (matejonnet) [inline model/src/main/java/org/jboss/pnc/model/User.java]: Any idea why it's been working up to now ?

Comment (jbartece) [inline model/src/main/java/org/jboss/pnc/model/User.java]: I messed it up in the DB Review PR - it wasn't captured by tests

Comment (matejonnet) [inline model/src/main/java/org/jboss/pnc/model/User.java]: I see it used to be called `Users` which actually sounds better than `UserTable`.

**Pull request merged**